src/main/java/teammates/ui/controller/InstructorFeedbackEditCopyAction.java

```diff
@@ -29,14 +30,10 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         Assumption.assertNotNull("null fs name", originalFeedbackSessionName);
         Assumption.assertNotNull("null copied fs name", newFeedbackSessionName);
         
-        String currentPage = getRequestParamValue(Const.ParamsNames.CURRENT_PAGE);
+        // TODO remove currentPage parameter sent from the form submission
```

Comment:
It's not required for this pr, can be done in another issue to keep this pr smaller